packet_queue/command.py

```diff
@@ -45,30 +64,67 @@ def configure(rest_server=False):
       help=('proxy port for receiving all inbound traffic if -luser'
             'is specified'))
   parser.add_argument(
-      '-p', '--port', type=int, required=True,
+      '-p', '--port', type=int, action='append', dest='ports', required=True,
       help='flaky inbound/outbound traffic occurs on specified port')
+  parser.add_argument(
+      '-b', '--bandwidth', type=int, default=-1,
+      help='The bandwidth in bytes per second')
+  parser.add_argument(
+      '-d', '--delay', type=int, default=0,
+      help='The one way delay in seconds')
+  parser.add_argument(
+      '-B', '--buffer', type=int, default=-1,
+      help='The size of the buffer in bytes')
+  parser.add_argument(
+      '-L', '--loss', type=verify_packet_drop_range, default=0.0,
+      help='The packet drop ratio. must be in range 0.0-1.0')
+  parser.add_argument(
+      '-D', '--direction', default='inbound',
+      choices=['inbound', 'outbound'],
```

Comment:
This might be better described as "client" versus "server" instead of inbound versus outbound. Here's why.

"The direction of the original connection request" doesn't precisely capture what this parameter does. The reason you care about the direction of the original connection request is that the first packet is sent to a **known port**. That's really what matters, especially if you consider UDP application protocols that may not have the concept of a "connection."

So, "inbound" doesn't really mean "impair inbound packets", it means "impair *inbound* packets to a known port and impair outbound* packets from a known port. That's not easy to capture in a short word or phrase. But if people generally think of a "server" as a thing that listens on a known port, and a "client" as a thing that sends packets to a known port, then you can capture the behavior with those words.

I don't know what you name the parameter, though. "Side"?